QEfficient/transformers/models/gptj/__init__.py

```diff
@@ -0,0 +1,7 @@
+# -----------------------------------------------------------------------------
+#
+# Copyright (c)  2023-2024 Qualcomm Innovation Center, Inc. All rights reserved.
```

Comment:
Sorry small change, can you make this year range to `2024 `only pls?

Reply:
will make these changes in all files in seperate PR.